android/quest/src/main/res/drawable/ic_grey_right_arrow.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="8dp"
+    android:height="14dp"
+    android:viewportWidth="8"
+    android:viewportHeight="14">
+  <path
+      android:pathData="M0.293,0.293C0.683,-0.098 1.317,-0.098 1.707,0.293L7.707,6.293C8.098,6.683 8.098,7.317 7.707,7.707L1.707,13.707C1.317,14.098 0.683,14.098 0.293,13.707C-0.098,13.317 -0.098,12.683 0.293,12.293L5.586,7L0.293,1.707C-0.098,1.317 -0.098,0.683 0.293,0.293Z"
+      android:fillColor="#919191"
+      android:fillType="evenOdd"/>
+</vector>
```

Comment:
I think we can use relevant compose `Icon` instead. You can define the tint color as desired.

Reply:
Removed this resource